tests/test_integration_2.py

```diff
@@ -158,7 +152,7 @@ def display_data(n_intervals):
 
         self.startServer(app=app)
 
-        start_button = self.wait_for_element_by_css_selector("#start")
+        self.wait_for_element_by_css_selector("#start")
```

Comment:
non-blocking, but we can just remove this line.

Reply:
https://github.com/plotly/dash-core-components/pull/777/commits/2ce39da054dbfd68bc5b24f3501a51f2f51d2d9e